crates/swc_ecma_preset_env/data/core-js-compat/modules-by-versions.json

```diff
@@ -493,5 +493,16 @@
   ],
   "3.30": [
     "web.url.can-parse"
+  ],
+  "3.31": [
+    "es.string.is-well-formed",
+    "es.string.to-well-formed",
+    "esnext.function.metadata",
+    "esnext.object.group-by",
+    "esnext.promise.with-resolvers",
+    "esnext.symbol.is-registered-symbol",
+    "esnext.symbol.is-well-known-symbol",
+    "web.url-search-params.delete",
+    "web.url-search-params.has"
```

Comment:
The actual `core-js-compat` version is 3.32.1, but it seems you used 3.31 which does not contain some data.